tests/conftest.py

```diff
@@ -131,30 +143,102 @@ def samples_history_collection_accessor(mongo_database):
 @pytest.fixture
 def testing_samples(samples_collection_accessor):
     result = samples_collection_accessor.insert_many(TESTING_SAMPLES)
-    samples = list(samples_collection_accessor.find({"_id": {"$in": result.inserted_ids}}))
+    samples = list(samples_collection_accessor.find({FIELD_MONGODB_ID: {"$in": result.inserted_ids}}))
     try:
         yield samples
     finally:
         samples_collection_accessor.delete_many({})
 
 
+@pytest.fixture
+def testing_priority_samples(priority_samples_collection_accessor):
+    result = priority_samples_collection_accessor.insert_many(TESTING_PRIORITY_SAMPLES)
+    samples = list(priority_samples_collection_accessor.find({FIELD_MONGODB_ID: {"$in": result.inserted_ids}}))
+    try:
+        yield samples
+    finally:
+        priority_samples_collection_accessor.delete_many({})
+
+
+@pytest.fixture
+def testing_priority_samples_for_aldp(priority_samples_collection_accessor):
+    result = priority_samples_collection_accessor.insert_many(TESTING_PRIORITY_SAMPLES_FOR_ALDP)
+    samples = list(priority_samples_collection_accessor.find({FIELD_MONGODB_ID: {"$in": result.inserted_ids}}))
+    try:
+        yield samples
+    finally:
+        priority_samples_collection_accessor.delete_many({})
```

Comment:
I don't think this is required (nor `samples_collection_accessor.delete_many({})` on line 150 because the mongo_database fixture deletes the database after each test. I could be mistaken though!